solr/CHANGES.txt

```diff
@@ -396,7 +396,8 @@ Bug Fixes
 
 Other Changes
 ---------------------
-(No changes)
+* SOLR-15486: During node shutdown pausing of updates and waiting for in-flight update requests to finish
+  before closing cores is no longer SolrCloud specific. (Christine Poerschke, David Smiley)
```

Comment:
LOL all I did was agree with the idea.  But fine; I'm reviewing because I want to be deserving of my name being there ;-)